armsrc/iso14443b.c

```diff
@@ -1381,12 +1380,12 @@ static int Get14443bAnswerFromTag(uint8_t *response, uint16_t max_len, uint32_t
             if (AT91C_BASE_SSC->SSC_SR & (AT91C_SSC_ENDRX)) {
 
                 // primary buffer was stopped
-                if (AT91C_BASE_PDC_SSC->PDC_RCR == false) {
+                if (!AT91C_BASE_PDC_SSC->PDC_RCR) {
```

Comment:
Code style,   we are moving away from !expression  since its easy to miss its a negation.  
if (expression)
and 
if (expression == false)

is our current code style.

Reply:
Sure.